paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -93,10 +93,6 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
             onTransitionTarget(transitionTarget)
         }
 
-        if (savedInstanceState == null) {
-            viewModel.transitionToFirstScreenWhenReady()
-        }
```

Comment:
This can happen in the `init` of the view models. Moving it there also allows us to test this behavior in a simple view model test as opposed to using an `ActivityScenario`.